databricks/koalas/indexes.py

```diff
@@ -565,6 +567,8 @@ def name(self) -> Union[Any, Tuple]:
 
     @name.setter
     def name(self, name: Union[Any, Tuple]) -> None:
+        if not is_hashable(name):
```

Comment:
is there the same issue in series too?

Reply:
Oh yeah. Just fixed Series, too. Thanks!